### DISCUSSION TITLE
claude 通过代理服务器oneapi 调用返回401

### DISCUSSION THREAD

**Discussion by jyh583854536** [lobehub/lobehub#1679] [General Question] [answered]
### 💻 系统环境

Ubuntu

### 📦 部署环境

Docker

### 🌐 浏览器

Chrome

### 🐛 问题描述

claude通过代理服务器oneapi调用claude官方api时，头部没有Authorization，导致返回401
![Snipaste_2024-03-21_21-22-01](https://github.com/lobehub/lobe-chat/assets/33023978/e2e779e7-7dee-40a7-be88-394a47775f61)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
anthropic 的 provider 不是用的 openai 的接口规范，所以不支持 oneapi 的使用

**Comment by jyh583854536** [comment]
我用SemanticKernel 框架写了个客户端，调用Oneapi是可以的
![Snipaste_2024-03-22_11-54-11](https://github.com/lobehub/lobe-chat/assets/33023978/fb8360c0-fa99-49b1-abb2-2f66ad222ce1)

**Reply by arvinxx** [reply]
我是说 LobeChat 里的 anthropic provider 不支持 oneapi

**Comment by jyh583854536** [comment]
好
